recipes/cobra/run_test.py

```diff
@@ -0,0 +1,2 @@
+import cobra.test
```

Comment:
Shouldn't matter, but just in case can this become an executable? Namely `#!/usr/bin/env python` and appropriate permissions.